README.md

```diff
@@ -59,6 +59,24 @@ To test against a build made on this repository, you can use a link with this fo
 
 # Advanced debugging scenarios
 
+## Running the browser-interface
+
+In order to run browser interface in any platform follow the next instructions
+
+### Steps
+
+1. Open browser-interface with `Visual Studio Code`
+2. Make sure you have the `devcontainers` extension installed
+3. Make sure [Docker Desktop](https://www.docker.com/) is running
+4. `At Visual Studio Code` you will find a green button at the bottom-left corner of your screen, if you press it, a menu will appear and you have to press `Reopen in Container`
```

Comment:
`At Visual Studio Code` you will find a green button at the bottom-left corner of your screen (it has _"Open a Remote Window"_  tooltip),